code/controllers/subsystem/minimap.dm

```diff
@@ -689,6 +689,11 @@ SUBSYSTEM_DEF(minimaps)
 	if(faction == FACTION_NEUTRAL && isobserver(user))
 		faction = allowed_flags == MINIMAP_FLAG_XENO ? XENO_HIVE_NORMAL : FACTION_MARINE
 
+	if(is_xeno && xeno.hive.see_humans_on_tacmap)
+		allowed_flags = MINIMAP_FLAG_ALL
+		faction = FACTION_NEUTRAL
```

Comment:
```suggestion
```
Why did you think this was necessary?

Reply:
I copypasted this from observer tacmap :sob: